src/main/kotlin/com/wandera/affectedmodulesdetector/output/LogPrintOutput.kt

```diff
@@ -27,13 +27,13 @@ object LogPrintOutput {
     }
 
     private fun printSection(title: String, content: String) {
-        println(
-        """
-        
-        ------------------ $title ------------------
-        $content
-        ------------------ $title ------------------
-        """.trimIndent()
-        )
+        val builder = StringBuilder().apply {
+            appendLine()
+            appendLine("------------------ $title ------------------")
+            appendLine(content)
+            appendLine("------------------ $title ------------------")
+        }
+
+        println(builder.toString())
```

Comment:
Ok, I just learned that `trimIndent()` also `also removes the first and the last lines if they are blank` (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-indent.html) for whatever weird reason. I wouldn't complicate that in such case, sorry for that.